espnet2/train/trainer.py

```diff
@@ -513,6 +514,24 @@ def train_one_epoch(
                 all_steps_are_invalid = False
                 continue
 
+            if iiter == 1 and summary_writer is not None:
```

Comment:
This may try to add graph in every epoch, right?
If so, is it needed?

Reply:
Yes.

- If the graph is added only when 0epoch, the graph is not added when resuming from middle epochs.
- In this method, we can't judge whether it is the first epoch when resuming. We need to modify trainer.py a little bit.
- I think the graph creation is acceptable if it is only done once for each epoch.

Reply:
OK. I understand your point.

Reply:
If you found it time-consuming after merging, please tell me it.